### ISSUE TITLE
Typo?

### ISSUE THREAD

**Issue by mikkelpm** [kolesarm/ebci#1]
https://github.com/kolesarm/ebci/blob/cff28500da53d99e60c53872d8765419304b0cac/R/cv.R#L247
Should this just be m2, not sqrt(m2)?

**Comment by mikkelpm** [comment]
https://github.com/kolesarm/ebci/blob/cff28500da53d99e60c53872d8765419304b0cac/R/cv.R#L251
Also, this probably shouldn't be 0.05.

**Comment by kolesarm** [comment]
For line 247, it should indeed be `sqrt(m2)`, since `CVb` takes bound on worst-case bias as argument, and if `kappa==1`, then worst case bias is the same as square root of second moment. Compare:

```r
 c(cva(m2=4, kappa=1.001)$cv, cva(m2=4, kappa=1)$cv)
[1] 3.64543 3.64485
```

For line 251, thanks, fixed in commit [3be2189](https://github.com/kolesarm/ebci/tree/3be2189d8f1fa8a582f50a2930e80abe3fd19ac4)

**Comment by mikkelpm** [comment]
Sorry, I meant that there shouldn't be a square root in this part:
`x=c(0, sqrt(m2))`

**Comment by kolesarm** [comment]
Thanks, sorry for misunderstanding. Fixed in lastest commit [d4865e9](https://github.com/kolesarm/ebci/commit/d5f02edb2d83b82bb56cd514b6f90422e5d71a9d)
